src/doom/api.c

```diff
@@ -180,6 +180,14 @@ api_response_t API_RouteRequest(api_request_t req)
         }
         return API_CreateErrorResponse(405, "Method not allowed");
     }
+    else if (strcmp(path, "api/players") == 0)
```

Comment:
Added this as a new endpoint to avoid breaking the existing api.  Could be used in place of api/player though as in single player mode returns a list of 1 player